apps/video/test/marionette/info_view_test.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+var Video = require('./lib/video'),
+    assert = require('assert'),
+    Video_Options_View = require('./lib/video_options_view.js'),
+    Video_Info_View = require('./lib/video_info_view.js');
+
```

Comment:
nit: please use camel case for variable names, e.g., VideoOptionsView, VideoInfoView